netlify.toml

```diff
@@ -2,4 +2,10 @@
   environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "production", REACT_APP_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_SANITY_DATASET = "production", GATSBY_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LeqBFIaAAAAAPfLave4lUtLBKF2WYXinbaLNFqB", GATSBY_INTERCOM_APP_ID = "kn5di10s" }
 
 [context.deploy-preview]
-  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_DEPLOY_PREVIEW = "true", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging",GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-hambach-1", REACT_APP_LEDGER_REST_ENDPOINT = "http://hambach.regen.network:1317", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Hambach Testnet", REACT_APP_BLOCK_EXPLORER = "https://hambach.regen.aneka.io" }
+  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging", GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-redwood-1", REACT_APP_LEDGER_REST_ENDPOINT = "http://redwood.regen.network:1317", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Redwood Testnet", REACT_APP_BLOCK_EXPLORER = "https://redwood.regen.aneka.io" }
```

Comment:
After this PR is merged into master, I'll create `redwood` and `hambach` branches.
On `redwood`, the current config for deploy previews won't change while on `hambach`, I'll update it to be the one from `context.hambach` (line 8)